_posts/2021-05-20-implementing-custom-facts-for-foreman-discovery.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: Implementing Custom Facts for Foreman Discovery
```

Comment:
Oh come on, this is a boring title. I want a clickbait article! Something juicy!!!

Automatic discovery of cluster of Intel NUC machines

Or something like that :-)